### DISCUSSION TITLE
Add events for file classification inputs

### DISCUSSION THREAD

**Discussion by Olie-Chanz** [formkit/formkit#871] [General]
We just migrated from Vue 2 to Vue 3 and that also means migrating our vue-formulate to formkit.

There are certain events that we used to show progress / loading state when uploading a file on a file input field where as: `@file-upload-progress`, `@file-upload-complete`, `@file-upload-error`, `@file-removed`. But these seems to be not available in FormKit. Is this something planned in the future?

**Comment by justin-schroeder** [comment]
We are planning to have a much more comprehensive file uploader in the pro package in the future. The `file` input itself (in FormKit) is intended to more closely mirror a native file input rather than the highly-opinionated file input in Vue Formulate (which was awesome, but perhaps was a bit too opinionated for the "base" file input).
